plugins/modules/zos_job_output.py

```diff
@@ -318,7 +341,12 @@
           "code": 0,
           "msg": "CC 0000",
           "msg_code": "0000",
-          "msg_txt": ""
+          "msg_txt": "",
+          "steps": [
+            { "stepid": "STEP0001",
```

Comment:
If you can update stepid to stepname it corresponds to SDSF JS panel names

Reply:
Changed in code, and in documentation (zos_job_output, zos_job_query)

Reply:
@richp405 some of the samples are not updated but the code is, I think I will wait to review this because [PR 260](https://github.com/ansible-collections/ibm_zos_core/pull/260/files) has a number of comments and both this PR and 260 share `module_utils/job.py`

Reply:
I found 3 instances in zos_job_submit, and grepped the modules directory, but did not find others.